scripts/workers/code_upload_submission_worker.py

```diff
@@ -461,6 +461,46 @@ def read_job(api_instance, job_name):
     return api_response
 
 
+def cleanup_submission(
+    api_instance,
+    evalai,
+    job_name,
+    submission_pk,
+    challenge_pk,
+    phase_pk,
+    stderr,
+    message,
+):
+    """Function to update status of submission to EvalAi, Delete corrosponding job from cluster and messaage from SQS.
+    Arguments:
+        api_instance {[AWS EKS API object]} -- API object for deleting job
```

Comment:
@savish28 please add description for all params if adding docstrings

Reply:
@Ram81 Updated.